### PR TITLE
Ant task check dependencies

### PR DESCRIPTION
**Pull request by pazi146**
- checks for unused dependencies

### CODE REVIEW

File: plugins/org.eclipse.fordiac.ide.globalconstantseditor.ui.tests/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -9,7 +9,6 @@ Bundle-ActivationPolicy: lazy
 Require-Bundle: org.eclipse.fordiac.ide.globalconstantseditor.ui,
  org.eclipse.xtext.testing,
  org.eclipse.xtext.xbase.testing,
- org.eclipse.xtext.xbase.junit,
```
Comment (ernstblechaPT) [inline plugins/org.eclipse.fordiac.ide.globalconstantseditor.ui.tests/META-INF/MANIFEST.MF]: I think this should be at least a seperate commit

Comment (pazi146) [reply plugins/org.eclipse.fordiac.ide.globalconstantseditor.ui.tests/META-INF/MANIFEST.MF]: thanks for catching that

**Comment by ernstblechaPT** [comment]
I have tested the ant-task with a new, empty project (no instances in appliction, no system configured):

` Unused dependencies detected: convert, signalprocessing, net, utils, math, devices, segments, reconfiguration, resources`

I don't see why the rest of the linked libraries are not listed as well?

**Comment by oberlehner** [comment]
` iec61131-3` is also missing.

**Comment by pazi146** [comment]
Some standard libraries depend on others:

- `convert` depends on `core`
- `events` depends on `iec61131-3`
- `iec61131-3` depends on `core`
- `net` depends on `core`
- `rtevents` depends on `core` and `events`
- `utils` depends on `core`

Result: `core`, `events` and `iec61131-3` will not show as unused when including all standard libraries

**Review by ernstblechaPT — APPROVED** [review]

**Pull request merged**